mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java

```diff
@@ -203,10 +203,7 @@ protected Boolean doInBackground(String... strings) {
                         Log.d(TAG, "Socket connected");
                         s.close();
                         return true;
-                    } catch (MalformedURLException e) {
-                        Log.e(TAG, e.getMessage());
-                        return false;
-                    } catch (IOException e) {
+                    } catch (Exception e) {
```

Comment:
I'm fine with the general exception handling then.  The signing requirement can be done by amending your last commit ( https://help.github.com/articles/changing-a-commit-message/ ) and adding one extra line like ```Signed-off-by: Joe Smith <joe.smith@email.com> (github: github_handle)``` .   I'm going to test this today, thanks!